en/docs/setup/migrating-what-has-changed.md

```diff
@@ -375,4 +375,13 @@ SMS provider.
     ```toml
     [authentication.authenticator.sms_otp.parameters]
     UseInternalErrorCodes = false
-    ```
\ No newline at end of file
+    ```
+## New User Store Managers with built in unique id support. 
+
+New user store managers with inbuilt unique id support will provide the capability to enhance the user store operations 
+that were initially architecturally impossible. Such as username rename functionality and significant performance 
```

Comment:
Shall we rephrase this? 

These operations include **username rename** functionality and a **significant performance increase**.